doc/classes/Basis.xml

```diff
@@ -340,94 +370,124 @@
 		</constant>
 		<constant name="FLIP_Y" value="Basis(1, 0, 0, 0, -1, 0, 0, 0, 1)">
-			The basis that will flip something along the Y axis when used in a transformation.
+			The basis that will flip another basis along its [member y] axis when transformed (multiplied) by it.
 		</constant>
 		<constant name="FLIP_Z" value="Basis(1, 0, 0, 0, 1, 0, 0, 0, -1)">
-			The basis that will flip something along the Z axis when used in a transformation.
+			The basis that will flip another basis along its [member z] axis when transformed (multiplied) by it.
 		</constant>
 	</constants>
 	<operators>
 		<operator name="operator !=">
 			<return type="bool" />
 			<param index="0" name="right" type="Basis" />
 			<description>
-				Returns [code]true[/code] if the [Basis] matrices are not equal.
-				[b]Note:[/b] Due to floating-point precision errors, consider using [method is_equal_approx] instead, which is more reliable.
+				Returns [code]true[/code] if the components of both [Basis] matrices are not equal.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Basis" />
 			<param index="0" name="right" type="Basis" />
 			<description>
-				Composes these two basis matrices by multiplying them together. This has the effect of transforming the second basis (the child) by the first basis (the parent).
+				Transforms (multiplies) this basis by the [param right] basis.
+				This is the operation performed between parent and child [Node3D]s.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Vector3" />
 			<param index="0" name="right" type="Vector3" />
 			<description>
-				Transforms (multiplies) the [Vector3] by the given [Basis] matrix.
+				Transforms (multiplies) this basis by the [param right] vector, returning a [Vector3].
+				[codeblocks]
+				[gdscript]
+				var my_basis = Basis(Vector3(1, 1, 1), Vector3(1, 1, 1), Vector3(0, 2, 5))
+				print(my_basis * Vector3(1, 2, 3)) # Prints (7, 3, 16)
+				[/gdscript]
+				[csharp]
+				var myBasis = new Basis(new Vector3(1, 1, 1), new Vector3(1, 1, 1), new Vector3(0, 2, 5));
+				GD.Print(my_basis * new Vector3(1, 2, 3)); // Prints (7, 3, 16)
+				[/csharp]
+				[/codeblocks]
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Basis" />
 			<param index="0" name="right" type="float" />
 			<description>
-				This operator multiplies all components of the [Basis], which scales it uniformly.
+				Multiplies all components of the [Basis] by the given [float]. This affects the basis' scale proportionately.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Basis" />
 			<param index="0" name="right" type="int" />
 			<description>
-				This operator multiplies all components of the [Basis], which scales it uniformly.
+				Multiplies all components of the [Basis] by the given [int]. This affects the basis' scale proportionately.
```

Comment:
"Uniformly" is *the* term for this operation, not "proportionately". "basis uniform scale" on Google gives useful search results, "basis proportionate scale" gives no useful results.